### DISCUSSION TITLE
Can I add table filters information above table part?

### DISCUSSION THREAD

**Discussion by PetroZaburko** [pxlrbt/filament-excel#164] [Q&A] [answered]
### Package
pxlrbt/filament-excel

### Package Version
2.2

### Laravel Version
10.31.0

### Livewire Version
v3.1.0

### PHP Version
8.2.2

### Question
1. Can I add some information from table filters above table part, like this:
![Screenshot from 2023-12-28 15-58-52](https://github.com/pxlrbt/filament-excel/assets/56628382/1836cf41-cdae-4873-b5d7-54c0014c0d1b)

2. Can I made heading bold?

**Comment by pxlrbt** [comment, marked as answer]
Sorry, I don't think there is an easy way to add your own rows to the table header

**Comment by PetroZaburko** [comment]
Thank you very much, it's ok
